### PR TITLE
docs: add animations documentation for the beta

### PR DESCRIPTION
**Pull request by fbasso**

### CODE REVIEW

File: demo/src/app/pages/animations/animations.component.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import {Component} from '@angular/core';
+import {Snippet} from '../../shared/code/snippet';
+
+@Component({templateUrl: './animations.component.html'})
+export class AnimationsPage {
+  rightExample = Snippet(
```
Comment (maxokorokov) [inline demo/src/app/pages/animations/animations.component.ts:6]: I think you forgot to remove this one

File: demo/src/app/pages/animations/animations.component.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<ngbd-page-wrapper pageTitle="Animations">
+    <ngb-alert type="info" [dismissible]="false">
+        <h4 class="d-flex align-items-center">
+            Beta testing starting with <span
+                class="ml-1 badge badge-info align-middle font-weight-normal">7.0.0-beta.1</span>
+        </h4>
+        Animations will coming with ng-bootstrap 7. They are already available in the beta 7, so feel free to use it and
```
Comment (jnizet) [inline demo/src/app/pages/animations/animations.component.html:7]: will come? will be coming?

File: demo/src/app/pages/animations/animations.component.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    <p>
+        To turn off animations globally, inject <code>NgbConfig</code> at your application root and change the
+        <code>animation</code>
+        flag. For example:
+    </p>
+
+    <ngbd-code lang="typescript" [snippet]="config"></ngbd-code>
+
+    <p>
+        You can also choose to disabled them by widget type, by changing the widget configuration:
```
Comment (jnizet) [inline demo/src/app/pages/animations/animations.component.html:32]: disabled -> disable

File: demo/src/app/pages/animations/animations.component.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        is not directly connected to the nav widget, no animation will be run either. Two workarounds available:</p>
+    <ul>
+        <li>Do it yourself <a href="https://angular.io/guide/route-animations">the Angular way</a></li>
+        <li>Set the nav panels the usual way, and use the <a
+                href="https://angular.io/guide/router#accessing-query-parameters-and-fragments"><code>ActivatedRoute</code></a>
+            to initialise the active tab with the url information.</li>
+    </ul>
+    <ngbd-page-header title="Found an issue ?" fragment="issues"></ngbd-page-header>
+    <p>
+        Playing with animations has been an hard work, as we had to manage a lots of internal things asynchronously when
```
Comment (jnizet) [inline demo/src/app/pages/animations/animations.component.html:70]: an hard -> a hard
a lots -> a lot

**Comment by fbasso** [comment]
Thanks for the review @jnizet ! I will fix the typos.

**Comment by benouat** [comment]
@fbasso as we are shifting animations to v8, would you mind updating quickly the couple references we have in this doc page, so we could merge it for v7 ?

**Comment by fbasso** [comment]
@benouat, sure, I'll have a look.

**Comment by fbasso** [comment]
I changed the version to 8, and rebased on master to get the latest changes.

Regarding the information, I'm not sure about this:

> More generally, changing the widget state with a model binding will not trigger an animation. You will need to run the widget api instead. For example for the nav, use select to change the nav programmatically.

We will need to check this point (to see if it's applicable to every widget) after their review and integration in the animation branch.

**Comment by maxokorokov** [comment]
Fixed some typos, added `npm install @ng-bootstrap/ng-bootstrap@next` snippet, removed some phrases that we've actually fixed :) Will merge when green.

**Pull request merged**